### DISCUSSION TITLE
When can I use rocm on windows?

### DISCUSSION THREAD

**Discussion by ttyyzz34** [ROCm/ROCm#2003] [Q&A]
Can anyone tell me when rocm will be available on windows？

**Comment by Odin234** [comment]
It's a few years old question.
And AMD will just completely ignore this kind of rocm question.

Don't bother to ask.
Switch to other gpu manufacturer.

**Comment by lanwatch** [comment]
Likely 5.6
https://videocardz.com/newz/amd-working-on-rocm-support-for-windows-os

**Comment by saadrahim** [comment]
No public announcement on a Windows launch has been made by AMD. Please do not assume anything from unofficial sources.

**Comment by Lumpsack** [comment]
Might already be too late for the AI wave - a real shame, with the VRAM they put in the consumer cards they could have cleaned up, but now the advice is "just buy nvidia" instead.

**Reply by Lumpsack** [reply]
5.6 teaser out (coming in Autumn) and no word of Windows

**Comment by lanwatch** [comment]
This is already available since v5.6 (July).

https://rocm.docs.amd.com/en/latest/release/windows_support.html

**Reply by ccbadd** [reply]
There is no v5.6 for Windows right now, only Linux.  V5.5 for Windows is pretty useless too right now as other app like pytorch don't support the windows version yet.

"Part of the AMD ROCm Platform, the HIP SDK is now available on Windows. 5.5 was released on July 27th, 2023. Please read [About ROCm](https://rocm.docs.amd.com/en/latest/rocm.html) to learn more about the HIP SDK."

It is not up to date with the features of the Linux 5.6 version.

**Reply by lanwatch** [reply]
I see, I hadn't realized MIOpen & Pytorch don't work yet. For my "normal" CUDA/HIP codes it seems to work ok-ish. It was a pleasant surprise last week, actually. I was not tracking this too closely.
